### PR TITLE
Added changelog message regarding webservice timeout

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 * Added ability to override the default zone(s) used by JES via the config structure by setting `genomics.default-zones` in the JES configuration
 * Added support for new WDL functions:
   * `length: (Array[X]) => Integer` - report the length of the specified array 
+* The cromwell server timeout is now configurable via the config key `webservice.timeout`, with the same default of `5s`.
```
Comment (cjllanwarne) [inline CHANGELOG.md]: Is this in `reference.conf`?

Comment (cjllanwarne) [inline CHANGELOG.md]: What does "server timeout" mean, specifically?

**Comment by cjllanwarne** [comment]
👍

File: CHANGELOG.md (hunk 1 above)
Comment (mcovarr) [inline CHANGELOG.md]: Can this be `5 seconds`?

**Comment by mcovarr** [comment]
:+1:

File: CHANGELOG.md (hunk 1 above)
Comment (kshakir) [inline CHANGELOG.md]: I'll change it to binding timeout.

**Pull request merged**